resources/views/info.blade.php

```diff
@@ -7,7 +7,9 @@
 
 <div
     data-tippy-content="{{ $tooltip }}"
-    class="inline-block cursor-pointer {{ $large ? 'p-1.5' : 'p-1' }} transition-default rounded-full bg-theme-primary-100 text-theme-primary-600 dark:bg-theme-secondary-800 dark:text-theme-secondary-600 hover:text-white hover:bg-theme-primary-700 dark:hover:text-theme-secondary-800 dark:hover:bg-theme-secondary-600 {{ $class }}"
+    aria-label="{{ $tooltip }}"
+    class="inline-block cursor-pointer {{ $large ? 'p-1.5' : 'p-1' }} transition-default rounded-full bg-theme-primary-100 text-theme-primary-600 dark:bg-theme-secondary-800 dark:text-theme-secondary-600 hover:text-white hover:bg-theme-primary-700 dark:hover:text-theme-secondary-800 dark:hover:bg-theme-secondary-600 outline-none focus-visible:ring-2 focus-visible:ring-theme-primary-500 {{ $class }}"
+    tabindex="0"
```

Comment:
this makes info tooltip interactable by keyboard `tabindex="0"` and it will also be announced to screen-readers via `aria-label="..."`.